sql/analyzer/resolve_columns.go

```diff
@@ -481,6 +481,34 @@ func indexColumns(a *Analyzer, n sql.Node, scope *Scope) map[tableCol]indexedCol
 			idx++
 		}
 	}
+	indexSchemaForDefaults := func(column *sql.Column, order *sql.ColumnOrder, sch sql.Schema) {
+		tblSch := make(sql.Schema, len(sch))
+		copy(tblSch, sch)
+		if order == nil {
+			tblSch = append(tblSch, column)
+		} else if order.First {
+			tblSch = append(sql.Schema{column}, tblSch...)
+		} else { // must be After
```

Comment:
No error case for invalid after column. Make sure there is a test for this.

Reply:
(should throw an error because the after column doesn't exist)

Reply:
Ohhh fantastic catch!